sdk/storage/Azure.Storage.DataMovement/tests/LocalDirectoryStorageResourceTests.cs

```diff
@@ -14,7 +14,7 @@
 
 namespace Azure.Storage.DataMovement.Tests
 {
-    public class LocalDirectoryStorageResourceTests : StorageTestBase<StorageTestEnvironment>
+    public class LocalDirectoryStorageResourceTests : DataMovementTestBase
```

Comment:
If this new parent class is only due to things that can be extracted out per earlier comment, we should revert this.

Reply:
probably can talk about this in the above questions I had about removing the disposing local directory.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/35408